File: C#/prog_lab_6/circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace prog_lab_6
{
    internal class circle
    {
        private point point;
        private double radius;
        public circle()
        {
            point = new point();
        }
        public void vvod()
        {
            Console.WriteLine("Введите координаты центра окружности.");
            point.vvod();
            Console.WriteLine("Введите длинну радиуса окружности.");
            while (!double.TryParse(Console.ReadLine(), out radius))
                Console.WriteLine("Радиус введен неверно.");
        }
        public void vivod()
        {
            Console.Write($"Центр окружности: ");
            point.vivod();
            Console.Write($" Радиус = {radius}");
            Console.WriteLine();
        }
        public double square()
        {
            return radius*radius*Math.PI;
        }
        public double per()
        {
            return 2*Math.PI*radius;
        }
    }
}

File: Java/prog_lab_7/src/prog_lab_7/circle.java
package prog_lab_7;
import java.util.Scanner;

public class circle {
	point center;
	double radius;
	
	public circle() {
		center = new point();
	}
	
	public void vvod() {
		this.center.vvod();
		Scanner in = new Scanner(System.in);
		this.radius = in.nextDouble();
	}
	
	public void vivod()	{
		System.out.printf("(%f; %f;   %f)\n", this.center.getx(), this.center.gety(), this.radius);
	}

	public double square() {
            return radius*radius*Math.PI;
	}
	
	public double per() {
            return 2*Math.PI*radius;
	}
}